agent/util-scripts/pbench-sysinfo-dump

```diff
@@ -66,11 +66,17 @@ function collect_block {
 function collect_sos {
 	name=$(hostname -f)-pbench
 	debug_log "[$script_name]collecting sosreport"
-	sos_ver=`rpm -q sos | awk -F- '{print $2}' | awk -F. '{print $1}'`
+	read sos_ver sos_ver_minor <<< `rpm -q sos | awk -F- '{print $2}' | awk -F. '{print $1" "$2}'`
 	if [[ "$sos_ver" == "3" ]]; then
 		quiet="--quiet"
 		block="-o block"
 		processor="-o processor"
+		# ref for plugin collectd: https://github.com/sosreport/sos/pull/866
+		if [[ "$sos_ver_minor" == "4" ]]; then
```

Comment:
okay thanks. Let me know if the latest push looks good?

Hope we're not looking for >=3 major, which might have breaking changes.